lib/media/track/remotetrackpublication.js

```diff
@@ -89,6 +95,10 @@ class RemoteTrackPublication extends TrackPublication {
           this.track._setSwitchedOff(signaling.isSwitchedOff);
         }
       }
+      if (priority !== signaling.priority) {
+        priority = signaling.priority;
+        this.emit('publishPriorityChanged', priority);
```

Comment:
I suggest we should pass oldPriority to the even (instead of / along with) new priority - But this need not hold of this PR

Reply:
@makarandp0 See my previous comment.